### ISSUE TITLE
Template store?

### ISSUE THREAD

**Issue by alexellis** [pmlopes/openfaas-svm-vertx#3]
Hi @pmlopes would you consider submitting to the template store?

https://www.openfaas.com/blog/template-store/

Alex

**Comment by pmlopes** [comment]
@alexellis Yes, I've been working on removing the sharp edges (relying on snapshot artifacts and duplicated interfaces). I'm now going a testing phase and will propose it soon
